### PR TITLE
Setup feature

### PR DESCRIPTION
**Pull request by Alusp**
## I implemented the following in this PR:

- [ ] Setup linters
- [ ] Setup Gitflow 
- [ ] Create a new app. with a meaningful name.
- [ ] Initialize my project with Git.
- [ ] Setup Postgres database set up.
- [ ] Build my project schema.
- [ ] Create and run the necessary migration files.
- [ ] Table and column names match the ERD diagram.
- [ ] note: photo for users table should be a link to an image
- [ ] Foreign keys should be included.
- [ ] All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

File: db/migrate/20221125083532_add_user_ref_to_comments.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, foreign_key: true
```
Comment (VICTOR-CHUKWU) [inline db/migrate/20221125083532_add_user_ref_to_comments.rb:3]: - [ ] according to project requirement, kindly reference user as author i.e `t.references :author, foreign_key: { to_table: 'users' } `

File: db/migrate/20221125083555_add_user_ref_to_likes.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, foreign_key: true
```
Comment (VICTOR-CHUKWU) [inline db/migrate/20221125083555_add_user_ref_to_likes.rb:3]: - [ ]  according to project requirement, kindly reference user as author i.e `t.references :author, foreign_key: { to_table: 'users' } `

File: db/migrate/20221125083625_add_user_ref_to_posts.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, foreign_key: true
```
Comment (VICTOR-CHUKWU) [inline db/migrate/20221125083625_add_user_ref_to_posts.rb:3]: - [ ]  according to project requirement, kindly reference user as author i.e `t.references :author, foreign_key: { to_table: 'users' } `

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Alusp  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alusp** [comment]
Thanks for the review @VICTOR-CHUKWU, I have resolve the requested changes.

**Review by uwadonat — APPROVED** [review]
## Hi @Alusp,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- Well organized and clean project ✔️
approval2

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Alusp** [comment]
Thanks for the review @uwadonat

**Pull request merged**